integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageDorisTest.java

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.integration.test.web.ui;
+
+import com.datastrato.gravitino.Catalog;
+import com.datastrato.gravitino.NameIdentifier;
+import com.datastrato.gravitino.client.GravitinoAdminClient;
+import com.datastrato.gravitino.client.GravitinoMetalake;
+import com.datastrato.gravitino.integration.test.container.ContainerSuite;
+import com.datastrato.gravitino.integration.test.container.DorisContainer;
+import com.datastrato.gravitino.integration.test.util.AbstractIT;
+import com.datastrato.gravitino.integration.test.util.ITUtils;
+import com.datastrato.gravitino.integration.test.util.JdbcDriverDownloader;
+import com.datastrato.gravitino.integration.test.web.ui.pages.CatalogsPage;
+import com.datastrato.gravitino.integration.test.web.ui.pages.MetalakePage;
+import com.datastrato.gravitino.integration.test.web.ui.utils.AbstractWebIT;
+import com.datastrato.gravitino.rel.Column;
+import com.datastrato.gravitino.rel.expressions.NamedReference;
+import com.datastrato.gravitino.rel.expressions.distributions.Distributions;
+import com.datastrato.gravitino.rel.expressions.sorts.SortOrder;
+import com.datastrato.gravitino.rel.types.Types;
+import com.google.common.collect.Maps;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@Tag("gravitino-docker-it")
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class CatalogsPageDorisTest extends AbstractWebIT {
+  MetalakePage metalakePage = new MetalakePage();
+  CatalogsPage catalogsPage = new CatalogsPage();
+
+  private static final ContainerSuite containerSuite = ContainerSuite.getInstance();
+  protected static GravitinoAdminClient gravitinoClient;
+  private static GravitinoMetalake metalake;
+
+  protected static String gravitinoUri = "http://127.0.0.1:8090";
+  protected static String dorisUri = "http://127.0.0.1:9030";
+
+  private static final String CATALOG_TABLE_TITLE = "Schemas";
+  private static final String SCHEMA_TABLE_TITLE = "Tables";
+  private static final String TABLE_TABLE_TITLE = "Columns";
+  private static final String METALAKE_NAME = "test";
+  private static final String CATALOG_TYPE_RELATIONAL = "relational";
+
+  private static final String DORIS_CATALOG_NAME = "catalog_doris";
+  private static final String DORIS_JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
+  public static final String USER_NAME = "root";
+  public static final String PASSWORD = "root";
+  public static final int FE_MYSQL_PORT = 9030;
```

Comment:
Remove it as it's unused.

Reply:
These are used for property settings of catalog creation

Reply:
<img width="1164" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/9dd2a307-e300-42ce-94f4-a67a904d2676">

Reply:
removed